test/SPIRV/AtomicCompareExchange_cl20.ll

```diff
@@ -71,7 +71,6 @@ entry:
 declare spir_func zeroext i1 @_Z30atomic_compare_exchange_strongPVU3AS4U7_AtomiciPU3AS4ii(i32 addrspace(4)*, i32 addrspace(4)*, i32) #1
 
 declare spir_func zeroext i1 @_Z28atomic_compare_exchange_weakPVU3AS4U7_AtomiciPU3AS4ii(i32 addrspace(4)*, i32 addrspace(4)*, i32) #1
-; RUN: llvm-as < %s | llvm-spirv -spirv-text | FileCheck %s
```

Comment:
This line was added here accidentally. First line of this file is exactly the same.